README.md

```diff
@@ -24,12 +23,19 @@ Exercism exercises in Haskell
 
 ## How to contribute
 
-### Reporting or fixing bugs
-Typical examples for a bug: A typo, a missing test case, an unclear or ambiguous
-problem description.
-- If you are unsure whether you have really found a bug [just ask](https://github.com/exercism/haskell/issues/new).
-- To report a bug you can [write an issue](#writing-an-issue).
-- If you already have a fix for it you may [write a pull request](#writing-a-pull-request).
+### Git and GitHub
+If you would like to contribute but lack experience with git and/or GitHub, try these resources:
+
+- [GitHub Help: Creating an issue](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
```

Comment:
the link target doesn't seem to match the text? creating an issue vs creating a pull request

Reply:
Thanks for spotting.